pkg/kudoctl/cmd/install/install.go

```diff
@@ -83,25 +105,29 @@ func verifyFrameworks(args []string) error {
 // Todo: needs testing
 // verifySingleFramework is the umbrella for a single framework installation that gathers the business logic
 // for a cluster and returns an error in case there is a problem
-func verifySingleFramework(name, previous string, gc *github.Client, k2c *kudo.Client) error {
-	// Get most recent ContentDir for selected Framework
-	content, err := gc.GetMostRecentFrameworkContentDir(name)
+func verifySingleFramework(name, previous string, r repo.FrameworkRepository, kc *kudo.Client) error {
+
+	i, err := r.IndexFile.Get(name, vars.PackageVersion)
 	if err != nil {
-		return errors.Wrap(err, "sorting most recent content dir")
+		return errors.Wrapf(err, "getting %s in index file", name)
 	}
 
-	if vars.PackageVersion != "" {
-		content, err = gc.GetSpecificFrameworkContentDir(name)
+	// checking if bundle exists locally already
+	bundleName := i.Name + "-" + i.Version
+	bundlePath := vars.RepoPath + "/" + bundleName
+	_, err = os.Stat(bundlePath)
+	if err != nil && os.IsNotExist(err) {
```

Comment:
inline this as:

`if _, err = os.Stat(bundlePath); err != nil && os.IsNotExist(err) {` // I think this should work, if the && throws the compiler for a loop don't worry about it.